src/microsim/lcmodels/MSLCM_LC2013.cpp

```diff
@@ -1076,6 +1076,9 @@ MSLCM_LC2013::prepareStep() {
             scaledDelta = deltaPosLat * myVehicle.getSpeed() / myVehicle.getLane()->getSpeedLimit();
         }
         myVehicle.setLateralPositionOnLane(oldPosLat + scaledDelta);
+        myVehicle.getLaneChangeModel().setSpeedLat(DIST2SPEED(scaledDelta));
+    } else if (!isChangingLanes()) {
+        myVehicle.getLaneChangeModel().setSpeedLat(0);
```

Comment:
should apply also to DK2008 (and other models people come up with, thus better placed in the base class.

Reply:
Does DK2008 have continuous lane changes and/or speedLat?

Why I put `setSpeedLat` in this if/else if clause:

- Each time step, only one `setSpeedLat`-call should be made, because each call updates myAccelerationLat. When simulating with sigma now, myAccelerationLat will not be correct.

Reply:
The continous lane change model is a layer above the individual lane change models so in principle it should work (even with SL2015 when not running with lateral resolution). However, it didn't until now #12144.
The point remains that the feature is not intended to be specific to LC2013.

Good point regarding `myAccelerationLat though`. I'll have to reconsider.

Reply:
Actually, we already had a similar problem with `sigmaLat` before: prepareStep updates speedLat and in the same step we may start a new lane change that updates speedLat again.